src/Parser/Core/Modules/Items/Legendaries.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ItemLink from 'common/ItemLink';
+import ITEM_QUALITIES from 'common/ITEM_QUALITIES';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SUGGESTION_IMPORTANCE from 'Parser/Core/ISSUE_IMPORTANCE';
+
+// Make sure to push this once we get to 1000!
+const MAX_LEGENDARY_ILVL = 970;
+
+class Legendaries extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  legendaries = [];
+  debug = false;
+
+  on_finished() {
+    const slots = this.combatants.selected._gearItemsBySlotId;
+    for(let index = 0; index < Object.keys(slots).length; index++) {
+      const item = slots[index];
+      if (item.quality === ITEM_QUALITIES.LEGENDARY) {
+        this.legendaries.push(item);
+      }
+    }
+  }
+
+  suggestions(when) {
+    for(const item of this.legendaries) {
+      if (this.debug) {
+        console.log(`Legendary ${item.id} is ${item.itemLevel}`);
+      }
+
+      when(item.itemLevel < MAX_LEGENDARY_ILVL).isTrue().addSuggestion((suggest) => {
```

Comment:
Could also be `when(item.itemLevel).isLessThan(MAX_LEGENDARY_ILVL)` but since we're not using the `actual` and `recommended` values it's not needed (just overall more used).